examples/material-ui-via-cdn/index.html

```diff
@@ -26,91 +13,32 @@
     />
     <!-- Icons to support Material Design -->
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <script type="importmap">
+      {
+        "imports": {
+          "react": "https://esm.sh/react@18.3.0",
+          "react-dom": "https://esm.sh/react-dom@18.3.0",
+          "react/jsx-runtime": "https://esm.sh/react@18.3.0/jsx-runtime",
+          "@mui/material": "https://esm.sh/@mui/material?external=react"
```

Comment:
Oh, interesting approach, makes sense to be able to use MUI X without duplication.